test/unit/UnitTestTimeNode.ts

```diff
@@ -110,4 +122,66 @@ describe('TimeNode Unit Tests', () => {
       assert.deepEqual(txs, ['0xe87529a6123a74320e13a6dabf3606630683c029']);
     });
   });
+
+  describe('handleDisconnections', () => {
+    it('detects Error  Disconnect', async () => {
+      const newconfig = await mockConfig();
+      if (!newconfig.providerUrl.includes('ws://') || !newconfig.providerUrl.includes('wss://')) {
```

Comment:
Please use the new `isWSConnection` method for this check.

Reply:
Fixed.